objects/oBoss1Head.object.gmx

```diff
@@ -63,6 +62,63 @@ wall2.image_yscale = 3;
 wheel = instance_create(x, y, oBoss1Wheel);
 
 sparkcounter = 0;
+
+//~TIMING~//
+enum BOSS1_ATTACKTIME {
```

Comment:
Enums should only be used for global-scope iterables; these would probably be better defined as constants (uppercase/underscores) since they're only used within the scope of this object and have specific, non-sequential numbers.

Reply:
I'll keep that in mind for the future